src/osx/cocoa/dataview.mm

```diff
@@ -2552,6 +2545,72 @@ void UpdateWithRow(int row)
     [m_OutlineView editColumn:column row:[m_OutlineView rowForItem:[m_DataSource getDataViewItemFromBuffer:item]] withEvent:nil select:YES];
 }
 
+#if wxUSE_DRAG_AND_DROP
+
+bool wxCocoaDataViewControl::EnableDropTarget(const wxDataFormatArray& formats)
+{
+    [m_OutlineView unregisterDraggedTypes];
+
+    if (formats.GetCount() > 0)
+    {
+        wxCFMutableArrayRef<CFStringRef> typesarray;
+        for (size_t i = 0; i < formats.GetCount(); ++i)
+            // formats.Item(i).AddSupportedTypes(typesarray);
```

Comment:
AFAIU, this adds osx carbon types instead of modern cocoa ones.

Reply:
NSStringPboardType is an alias for a NSString which is toll free bridged to a CFStringRef, the wxCFMutableArray can be cast to an NSArray* so this is fine

Reply:
@csomor, I mean that with just calling `formats.Item(i).AddSupportedTypes(typesarray);` in for-loop (w/o my swicth) DnD does not work at all. Am i doing something wrong?